docs/physics/montecarlo/initialization.ipynb

```diff
@@ -71,10 +71,39 @@
   },
```

Comment:
<p>It all looks really good -- I would suggest trying this without a <code>lambda</code> function, named functions should use <code>def</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Hi @smithis7 ,

due to my mistake, this PR was closed, 

please have a look at PR #2254, I have updated the Initialization notebook, in accordance to your suggestions

cheers